src/Loader.php

```diff
@@ -231,7 +232,7 @@ protected function sanitiseVariableValue($name, $value)
 
             // Unquoted values cannot contain whitespace
             if (preg_match('/\s+/', $value) > 0) {
-                throw new InvalidArgumentException('Dotenv values containing spaces must be surrounded by quotes.');
+                throw new SyntaxException();
```

Comment:
I still want the ability to pass a message here -  A `SyntaxException` could be a variety of errors - not just this one (though it is only this one at this time).